### PR TITLE
Update and change links on the SingleStore connector page

### PR DESCRIPTION
**Pull request by Joelg96**
## Description

* Removed language and country hard codes in links.

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/connector/singlestore.md [hunk 1]
```diff
@@ -25,10 +25,9 @@ To connect to SingleStore, you need:
 ## Configuration
 
 To configure the SingleStore connector, create a catalog properties file in
-`etc/catalog` named, for example, `example.properties`, to mount the
-SingleStore connector as the `example` catalog. Create the file with the
-following contents, replacing the connection properties as appropriate for your
-setup:
+`etc/catalog` named, for example, `example.properties`, to mount the SingleStore
```
[the remark this answers is not in the document]
Comment (Joelg96) [reply docs/src/main/sphinx/connector/singlestore.md:28]: Sorry, I did not change any wording, I only line-wrapped that paragraph.

[the remark this answers is not in the document]
Comment (Joelg96) [reply docs/src/main/sphinx/connector/singlestore.md:28]: I can change the wording though.

[the remark this answers is not in the document]
Comment (mosabua) [reply docs/src/main/sphinx/connector/singlestore.md:28]: Yeah .. lets update the wording in a separate commit. Maybe even do it for all other connectors where you find the "mount" mess

[the remark this answers is not in the document]
Comment (Joelg96) [reply docs/src/main/sphinx/connector/singlestore.md:28]: Okay. I can definitely do that!

**Comment by Joelg96** [comment]
@mosabua Going to close - I messed it up and somehow got ahead/behind. I'm going to make a new PR.

**Comment by mosabua** [comment]
I can help fix up if you want

**Pull request closed without merging**